### PR TITLE
EDU-164 Create UI language selection dialog

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-164

### CODE REVIEW

File: src/components/ui-language-dialog.less [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.UiLanguageDialog {}
```
Comment (laurentiu-ilici) [inline src/components/ui-language-dialog.less:1]: Is this really needed?

Comment (ahelmberger) [reply src/components/ui-language-dialog.less:1]: We always include it to have the Component part of SUITCSS there, it's compiled away anyway by the optimizer.

File: src/components/ui-language-dialog.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  const [selectedLanguage, setSelectedLanguage] = useState(language);
+  const [languagesToChoose, setLanguagesToChoose] = useState(createLanguagesToChoose(languageNameProvider, supportedLanguages, language));
+
+  useEffect(() => {
+    setLanguagesToChoose(createLanguagesToChoose(languageNameProvider, supportedLanguages, language));
+  }, [languageNameProvider, supportedLanguages, language]);
+
+  const handleOk = () => {
+    i18n.changeLanguage(selectedLanguage);
+    onClose?.();
```
[the remark this answers is not in the document]
Comment (ahelmberger) [reply src/components/ui-language-dialog.js:28]: Thankya

**Review by laurentiu-ilici — APPROVED** [review]

**Comment by adrianaluchian** [comment]
Looks super nice!

**Review by adrianaluchian — APPROVED** [review]

**Review by adrianaluchian — APPROVED** [review]

**Comment by ahelmberger** [comment]
> Looks super nice!

Thankyou!

**Review by adrianaluchian — APPROVED** [review]

**Review by adrianaluchian — APPROVED** [review]

**Pull request merged**